app/src/main/java/org/httpsknighthacks/knighthacksandroid/MainActivity.java

```diff
@@ -128,6 +126,13 @@ private void getImageAndTitles()
         mBgColors.add(getResources().getColor(R.color.colorHomePageCardPurple));
         activityList.add(Sponsors.class);
 
+        // TODO : add the correct image
```

Comment:
Could you please add the Profile stuff before everything else so that it appears first in the list instead of last?